frontend/src/lib/components/apps/editor/appUtils.ts

```diff
@@ -152,6 +152,26 @@ function findGridItemById(
 	return undefined
 }
 
+export function findEveryParentGridItem(app: App, id: string): string[] {
+	const parentGrids: string[] = []
+
+	function findParents(currentApp: App, currentId: string) {
+		const parentGrid = findGridItemParentGrid(currentApp, currentId)
+		if (parentGrid) {
+			parentGrids.push(parentGrid)
+
+			const parentId = parentGrid?.split('-')?.[0]
+
+			if (parentId) {
+				findParents(currentApp, parentId)
+			}
+		}
+	}
```

Comment:
that's super inefficient since findGridItemParentGrid already walk down the path. As discussed in discord, make a fork of findGridItemParentGrid that retains the path instead